aiogram/dispatcher/storage.py

```diff
@@ -444,7 +444,7 @@ def __contains__(self, item):
         return item in self._data
 
     def __str__(self):
-        readable_state = f"'{self.state}'" if self.state else "<default>"
+        readable_state = repr(str(self.state)) if self.state else "<default>"
```

Comment:
Converting to string is not needed here because self.state is Optional[str] and case with None already covered by this ternary-operator.
```suggestion
        readable_state = repr(self.state) if self.state else "<default>"
```